libs/auth/certs/prod/vx-cert-authority-cert.pem

```diff
@@ -0,0 +1,15 @@
+# A placeholder (and copy of certs/dev/vx-cert-authority-cert.pem)
+# until we have an official cert
```

Comment:
TIL re PEM file comments - openssl knows to ignore anything outside the `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`